routes/api/auth.js

```diff
@@ -0,0 +1,10 @@
+const router = require('express').Router()
+const authController = require('../../controllers/auth')
+
+router.route('/local-login')
```

Comment:
`/login` is preferable than `/local-login`, but pick whichever you like

Reply:
i chose local-login to differentiate routes as soon as we add social login, like facebook, google+, twitter.